h2/src/main/org/h2/command/Parser.java

```diff
@@ -2198,17 +2198,22 @@ private Prepared parseComment() {
             // if the db name is equal to the schema name
             ArrayList<String> list = Utils.newSmallArrayList();
             do {
+                if(database.getMode().allowEmptySchemaValuesAsDefaultSchema && readIf(DOT)) {
```

Comment:
1. Add a space after `if`.

2. Does Sybase really accepts leading dot(s) here? `COMMENT ON .PUBLIC.TEST.ID IS ''`

Reply:
Any comments? I can't test behavior of Sybase by myself.

Reply:
No it does not accept the leading dots. I wrote a test to show, that the code does not accept this.

Reply:
`COMMENT ON COLUMN .TEST.ID IS ''` still works in MSSQLServer mode, perhaps it should be rejected too?

Reply:
was also rejected, but not at the place, where intended.